### PR TITLE
COMMERCE-11030 Enable Request a Quote - Part 1 

### PR DESCRIPTION
**Pull request by smottal**
https://issues.liferay.com/browse/COMMERCE-11030

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/display/context/CommerceOrderContentDisplayContext.java [hunk 1]
```diff
@@ -127,6 +129,75 @@
  */
 public class CommerceOrderContentDisplayContext {
 
+	public CommerceOrderContentDisplayContext(
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/display/context/CommerceOrderContentDisplayContext.java:132]: add the configurationProvider to the other constructor method

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/display/context/CommerceOrderContentDisplayContext.java (hunk 1 above)
Comment (smottal) [reply modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/display/context/CommerceOrderContentDisplayContext.java]: Done

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:commerce-functional

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:close

**Pull request closed without merging**